model.py

```diff
@@ -27,7 +27,7 @@ def ge_layer(x_in, c, e=6, stride=1):
     else:
         y = x_in
         
-    x = layers.DepthwiseConv2D(depth_multiplier=e, kernel_size=(3,3), padding='same')(x)
+    x = layers.DepthwiseConv2D(kernel_size=(3,3), padding='same')(x)
```

Comment:
I think only changing this line might not get what is expected. When `stride = 1`, there is no expansion in the branch.